lib/xsl/partials-section.xsl

```diff
@@ -71,6 +71,32 @@
                         <section class="catalogo-section" id="{@n}">
                             <xsl:apply-templates/>
                         </section>
+
+                        <!-- Previous button
+                        Get the previous sibling (when previous is in the same section),
+                        or the last sibling of the previous sibling (when previous is in another section)
+                        See https://stackoverflow.com/a/11680802/446681 for the weird [1] syntax.
+                         -->
+                        <xsl:choose>
+                            <xsl:when test="current()/preceding::*[(@type='section' or @type = 'preface') and @n!=''][1]">
+                                <xsl:variable name="prev_section" select="(current()/preceding::*[(@type='section' or @type = 'preface') and @n!=''][1])/@n" />
+                                <a rel="prev" class="btn btn-default btn-sm" href="/{$url_path_prefix}catalogo/section_{$prev_section}">« Previous</a>
+                            </xsl:when>
+                            <xsl:otherwise>
+                                <a rel="prev" class="btn btn-default btn-sm disabled" href="#">« Previous</a>
+                            </xsl:otherwise>
+                        </xsl:choose>
+
+                        <!-- Next button -->
+                        <xsl:choose>
+                            <xsl:when test="empty(current()/following::*[(@type='section' or @type = 'preface') and @n!=''])">
+                                <a rel="next" class="btn btn-default btn-sm disabled" href="#">Next »</a>
+                            </xsl:when>
+                            <xsl:otherwise>
+                                <xsl:variable name="next_section" select="(current()/following::*[(@type='section' or @type = 'preface') and @n!=''])/@n" />
+                                <a rel="next" class="btn btn-default btn-sm" href="/{$url_path_prefix}catalogo/section_{$next_section}">Next »</a>
+                            </xsl:otherwise>
+                        </xsl:choose>
```

Comment:
This is the logic to generate the proper previous and next links. The links for previous are a bit trickier to generate since we want the last child of the previous sibling (rather than just the previous sibling). Hopefully the comments make that clear.